templates/Debian/10/etc/fail2ban/jail.conf.epp

```diff
@@ -167,7 +167,7 @@ fail2ban_agent = Fail2Ban/%(fail2ban_version)s
 # action_* variables. Can be overridden globally or per
 # section within jail.local file
 banaction = <%= $fail2ban::banaction %>
-banaction_allports = iptables-allports
+banaction_allports = <%= $fail2ban::banaction_allports %>
```

Comment:
This is good but could you could add a test.
Looks to be very similar to https://github.com/voxpupuli/puppet-fail2ban/blob/master/spec/classes/init_spec.rb#L308 for instance.